File: test/if/valid/multi_line.go
package main 

import "fmt" 

func main()   {
	if 1<2 {
		fmt.Printf("success \n")
		var a int = 0
		fmt.Printf("%d \n", a)
	}else{
		fmt.Printf("fail \n")
	}
	if 1>0 {
		fmt.Printf("success \n")
		var a int = 1
		fmt.Printf("%d \n", a)
	}else{
		fmt.Printf("fail \n")
	}
	if 1==1 {
		fmt.Printf("success \n")
		var a int = 2
		fmt.Printf("%d \n", a)
	}else{
		fmt.Printf("fail \n")
	}
	if 1!=2 {
		fmt.Printf("success \n")
		var a int = 3
		fmt.Printf("%d \n", a)
	}else{
		fmt.Printf("fail \n")
	}
	if 1!=2&&1<2 {
		fmt.Printf("success \n")
		var a int = 4
		fmt.Printf("%d \n", a)
	}else{
		fmt.Printf("fail \n")
	}
	if 1!=1||1<2 {
		fmt.Printf("success \n")
		var a int = 5
		fmt.Printf("%d \n", a)
	}else{
		fmt.Printf("fail \n")
	}
	if !(1==1) {
		fmt.Printf("fail \n")
	}else{
		fmt.Printf("success \n")
		var a int = 6
		fmt.Printf("%d \n", a)
	}
}

File: test/if/valid/multi_line.c
void main(){
    if (1 < 2){
        printf("success \n");
        int a = 0;
        printf("%d \n", a);
    } else{
        printf("fail \n");
    }
    
    if(1 > 0){
        printf("success \n");
        int a = 1;
        printf("%d \n", a);
    }else{
        printf("fail \n");   
    }

    if(1 == 1){
        printf("success \n");
        int a = 2;
        printf("%d \n", a);
    }else{
        printf("fail \n");
    }
    

    if (1 != 2){
        printf("success \n");
        int a = 3;
        printf("%d \n", a);
    } else{
        printf("fail \n");
    }


     if (1 != 2 && 1 < 2){
        printf("success \n");
        int a = 4;
        printf("%d \n", a);
    } else{
        printf("fail \n");
    }

    if (1 != 1 || 1 < 2){
        printf("success \n");
        int a = 5;
        printf("%d \n", a);
    } else{
        printf("fail \n");
    }
    

    if (!(1 == 1)){
        printf("fail \n");
    } else{
        
        printf("success \n");
        int a = 6;
        printf("%d \n", a);
    }
}